woocommerce/woocommerce-gateway-stripe
# fixed unhandled php error when invalid card is used on Pay Order page

Fixes #2196

## Changes proposed in this Pull Request:
This PR fixes an unhandled php error when an invalid card is used on Pay Order Page. The error handling was present on the code but it was in the wrong place. Just changed where the error is handled.

## Testing instructions
- With UPE disabled
- Add any product to card
- Go to checkout page
- Select card payment method
- Try to checkout with an invalid credit card on classic checkout 4000000000000002
- Go to pay My account > Orders > click on pay order
- Select Card as payment method
- Try to pay with invalid card again 4000000000000002
- The error `Notice: Undefined property: stdClass::$status in /var/www/html/public/wp-content/plugins/woocommerce-gateway-stripe/includes/class-wc-gateway-stripe.php on line 436` must not show up or be logged to debug.log

---

**Post merge**
